keyboards/tkc/candybar/lefty/keymaps/vial/vial.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "CandyBar Lefty",
+  "vendorId": "0x445A",
+  "productId": "0x2260",
+  "matrix": {
+      "rows": 4,
+      "cols": 17
+  },
+  "firmwareVersion": 0,
```

```suggestion
  "matrix": {
      "rows": 4,
      "cols": 17
  },
```
The other attributes are not used by Vial.